lib/scenic/statements.rb

```diff
@@ -36,7 +36,7 @@ def create_view(name, version: nil, sql_definition: nil, materialized: false)
       sql_definition ||= definition(name, version)
 
       if materialized
-        Scenic.database.create_materialized_view(name, sql_definition)
+        Scenic.database.create_materialized_view(name, sql_definition, materialized_no_data)
```

Comment:
I'd also like to use a keyword argument here. I think it will make the boolean more understandable.